ecs_fargate/ecs.tf

```diff
@@ -278,4 +279,23 @@ resource "aws_sns_topic_subscription" "cb_email_targets" {
   topic_arn = aws_sns_topic.cb[0].arn
   protocol  = "email"
   endpoint  = var.ecs_circuit_breaker_alert_email[count.index]
+}
+
+
+module "massgov_teamsalerts" {
```

Comment:
small thing, but this isn't necessarily for massgov

Reply:
renamed